thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/DetectionTaskRunner.java

```diff
@@ -138,6 +147,19 @@ private void runTask(DateTime windowStart, DateTime windowEnd) throws JobExecuti
     storeData(adOutputContext);
   }
 
+  private double calculateTrafficImpact (MetricTimeSeries metricTraffic, MergedAnomalyResultDTO mergedAnomaly) {
```

Comment:
Good catch. Currently, we cannot automatically determine this in the system. I will add a flag in AnomalyDetection to determine if we need to calculate traffic contribution.